src/main/java/TwitterApplication.java

```diff
@@ -0,0 +1,22 @@
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import resources.TwitterRequestHandler;
+
+public class TwitterApplication extends Application<ApplicationConfiguration>{
```

Comment:
much better name.  I normally wouldn't call it twitter app since it could expand beyond twitter but at least it's informative!